set.mm

```diff
@@ -74357,6 +74371,61 @@ of related elements (Contributed by Alexander van der Vekens,
 
+  ${
+    $d A a p $.  $d A a x y $.  $d F a x y $.  $d G a x y $.  $d H a $.
+    $d S a $.
+    fsplitfpar.h $e |- H = ( ( `' ( 1st |` ( _V X. _V ) )
+                        o. ( F o. ( 1st |` ( _V X. _V ) ) ) )
+                i^i ( `' ( 2nd |` ( _V X. _V ) )
+                       o. ( G o. ( 2nd |` ( _V X. _V ) ) ) ) ) $.
+    fsplitfpar.s $e |- S = ( `' ( 1st |` _I ) |` A ) $.
+    $( Merge two functions with a common argument in parallel.  Combination of
+       ~ fsplit and ~ fpar .  (Contributed by AV, 3-Jan-2024.) $)
+    fsplitfpar $p |- ( ( F Fn A /\ G Fn A )
+                -> ( H o. S ) = ( x e. A |-> <. ( F ` x ) , ( G ` x ) >. ) ) $=
+      ( va vy wfn wa cfv cop wceq wral wcel cvv a1i adantl vp ccom cv cmpt cres
+      c1st cid ccnv fsplit reseq1i eqtri fveq1i fvres eqidd weq id opeq12d elex
+      opex fvmptd eqtrd fveq2d co df-ov cmpo fpar adantr fveq2 simpr ovmpod wss
+      syl5eqr eqid fnmpt mprg ssv fnssres mp2an mpbir fvco2 sylan fvmpt 3eqtr4d
+      fneq1i ralrimiva wb cxp crn ralrimivva fnmpo fneq1d mpbird sylancl sylibr
+      syl cin wrex rneqi cima mptima df-ima rnmpt 3eqtr3i elinel2 simpl opelxpd
+      cab wi eleq1 ex rexlimiv abssi eqsstrid fnco syl3anc eqfnfv syl2anc ) DBK
+      EBKLZFCUBZABAUCZDMZXTEMZNZUDZOZIUCZXSMZYFYDMZOZIBPZXRYIIBXRYFBQZLZYFCMZFM
+      ZYFDMZYFEMZNZYGYHYLYNYFYFNZFMZYQYLYMYRFYLYMYFARXTXTNZUDZBUEZMZYRYMUUCOYLY
+      FCUUBCUFUGUEUHZBUEZUUBHUUDUUABAUIUJUKULSYKUUCYROXRYKUUCYFUUAMYRYFBUUAUMYK
+      AYFYTYRRUUARYKUUAUNAIUOZYTYROYKUUFXTYFXTYFUUFUPZUUGUQTYFBURYRRQZYKYFYFUSZ
+      SUTVATVAVBYLYSYFYFFVCYQYFYFFVDYLAJYFYFBBYAJUCZEMZNZYQFRXRFAJBBUULVEZOYKAJ
+      BBDEFGVFZVGUUFJIUOZLZUULYQOYLUUPYAYOUUKYPUUFYAYOOUUOXTYFDVHZVGUUOUUKYPOUU
+      FUUJYFEVHTUQTXRYKVIZUURYQRQYLYOYPUSZSVJVLVAXRCBKZYKYGYNOUUTXRUUTIRYRUDZBU
+      EZBKZUVARKZBRVKZUVCUUHUVDIRIRYRUVARUVAVMVNZUUHYFRQZUUISVOBVPZRBUVAVQZVRBC
+      UVBCUUEUVBHUUDUVABIUIUJUKZWDZVSSBFCYFVTWAYKYHYQOXRAYFYCYQBYDUUFYAYOYBYPUU
+      QXTYFEVHUQYDVMZUUSWBTWCWEXRXSBKZYDBKZYEYJWFXRFBBWGZKZUUTCWHZUVOVKUVMXRUVP
+      UUMUVOKZXRUULRQZJBPABPUVRXRUVSAJBBUVSXRXTBQZUUJBQLLYAUUKUSSWIAJBBUULUUMRU
+      UMVMWJWOXRUVOFUUMUUNWKWLXRUVCUUTXRUVDUVEUVCXRUUHIRPUVDXRUUHIRUUHXRUVGLUUI
+      SWEUVFWOUVHUVIWMUVKWNXRUVQUAUCZYROZIRBWPZWQZUAXGZUVOUVQUVBWHZUWECUVBUVJWR
+      UVABWSIUWCYRUDZWHUWFUWEIRYRBWTUVABXAIUAUWCYRUWGUWGVMXBXCUKUWEUVOVKXRUWDUA
+      UVOUWBUWAUVOQZIUWCYFUWCQYKUWBUWHXHYFRBXDYKUWBUWHYKUWBLZUWHYRUVOQZUWIYFYFB
+      BYKUWBXEZUWKXFUWBUWHUWJWFYKUWAYRUVOXITWLXJWOXKXLSXMUVOBFCXNXOXRYCRQZABPUV
+      NXRUWLABUWLXRUVTLYAYBUSSWEABYCYDRUVLVNWOIBXSYDXPXQWL $.
+
+    $d C a $.  $d O a $.  $d V a $.  $d W a $.
+    $( Express the function operation map ` oF ` by the functions defined in
+       ~ fsplit and ~ fpar .  (Contributed by AV, 4-Jan-2024.) $)
+    offsplitfpar $p |- ( ( ( F Fn A /\ G Fn A ) /\ ( F e. V /\ G e. W )
+                           /\ ( O Fn C /\ ( ran F X. ran G ) C_ C ) )
+                         -> ( O o. ( H o. S ) ) = ( F oF O G ) ) $=
```

Comment:
Thanks for formalizing that, that's nice !
Only: the choice of "O" is a bit strange (that letter is seldom used since it can be confused with zero); Mario chose "R" in https://us.metamath.org/mpeuni/df-of.html and that looks fine (it's indeed an operation, not a relation, but that's fine in my opinion).

Reply:
I have chosen "O" intentionally (for "operation" or "operator"), because I always get confused with using R as operation. ` A R B ` (binary relation, see ~df-br) and ` ( A R B ) ` (operation value, df-ov) can be easily confused, especially if embedded within a larger statement. Maybe we should choose a special connective symbol like `.+`, e.g.: `( .+ o. ( H o. S ) ) = ( F oF .+ G )`. There are already such cases in set.mm., see ~seqof or ~ghmplusg.

Reply:
Then maybe leave it like that. Color and font will help distinguish it from zero anyway.

Reply:
I replaced `O` by `.+`now.